elastic/apm-server
# [apmintegration] include java_attacher configuration

## Motivation/summary

enable configuration of java-attacher via apm integration package

## Checklist


## How to test these changes

install the integration and verify that configuration in the UI is reflected in the arguments used to start the java attacher. `log.level=debug` will output the command line arguments used to invoke the attacher.
